app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -439,25 +439,21 @@ def parse_base_item(item)
     def parse_image_name(image, image_ref)
       parts = %r{
         \A
-        (?:
-          (?<host>.*?)
-          (?::(?<port>.*?))?
-          /(?=.*/)
-        )?
-        (?<name>.*?)
-        (?::(?<tag>[^:]*?))?
+          (?:(?:(?<host>[^\.:\/]+\.[^\.:\/]+)|(?:(?<host2>[^:\/]+)(?::(?<port>\d+))))\/)?
+          (?<name>(?:[^:\/@]+\/)*[^\/:@]+)
+          (?:(?::(?<tag>.+))|(?:\@(?<shatag>sha256:.+)))?
         \z
       }x.match(image)
```

Comment:
@zeari is there a more authoritative place that describes what needs to be parsed and how?

Because we seem to have changed our mind on some, and what remains appears quite eclectic. (With the new "must have dots" rule, is this really parsing `host.com/name` as it should?)